static/app/views/explore/components/schemaHintsList.tsx

```diff
@@ -183,11 +184,9 @@ function SchemaHintsList({
         if (!isDrawerOpen) {
           openDrawer(
             () => (
-              <SchemaHintsDrawer
-                hints={filterTagsSorted}
-                exploreQuery={exploreQuery}
-                setExploreQuery={setExploreQuery}
-              />
+              <SchemaHintsProvider source={source}>
```

Comment:
I think instead of the provider we can maybe move the query variable below up and pass that in as `exploreQuery` instead, since it's the updated value we're pushing into the context (and thus the url) anyway